playbooks/templates/.env.j2

```diff
@@ -120,7 +120,7 @@ DISCORD_MENTION_USER_ID={{ discord_mention_user_id | default('') }}
 # For dev servers: {{ webportal_common_config.s3_backup_path }}_<server hostname>
 #
 # OPTIONAL: manually defined by user, can be blank
-S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path }}{{ '_' + inventory_hostname if 'webportals_prod' not in group_names else '' }}
+S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path }}/mongo
```

Comment:
`webportal_common_config.s3_backup_path` is for us currently set in `common.yml` in LastPass, which means with the change it will be shared between all `prod`, `dev`, `dev-cc` clusters. That's not ok, right?

We should either make it cluster specific like:  
`S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path }}/{{ portal_cluster_id }}-mongo` (I'm inclined to this)

or move this setting from `common.yml` to cluster yamls and make them manually unique between clusters.

And update `.env` vars on all servers in both cases.

Reply:
I also like the `S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path }}/{{ portal_cluster_id }}-mongo` approach. In that case we might actually drop the `-mongo` suffix. The env var is defined as a generic backup path, so it should be reusable.

I'll make the changes here.